CODE_OF_CONDUCT.md

```diff
@@ -14,10 +14,10 @@ appearance, race, religion, or sexual identity and orientation.
 Examples of behavior that contributes to creating a positive environment
 include:
 
-- Using welcoming and inclusive language
+- Using welcoming and [inclusive language](https://developers.google.com/style/inclusive-documentation)
```

Comment:
I like the idea of linking to an example, but is this the best one? Hard to map "inclusive documentation" back to "inclusive speech", which is more what this document is focused on.

(Separately, linking to this somewhere in the docs repo would still make sense)